config/sync/collection.settings.yml

```diff
@@ -25,6 +49,8 @@ transitions:
     validated:
       - moderator
       - rdf_entity-collection-facilitator
+  # In the following to-state, the facilitator will not actually have permission to execute the transition if there is
+  # not already a published version of the collection.
   validated:
```

Comment:
Yeah I know, this is how the access system has been implemented years ago. It is not in scope of this ticket to fix that. I am just moving the documentation here from its original location in the collection `config/install` legacy config so it doesn't get lost.

Reply:
Created a followup to tackle this issue: [ISAICP-6154: Rework workflow transitions to be permission based rather than role based](https://citnet.tech.ec.europa.eu/CITnet/jira/browse/ISAICP-6154).